Uyanda.Coffee.Application/Features/BeverageManagement/Persistence/IBeverageAccessor.cs

```diff
@@ -12,6 +12,6 @@ public interface IBeverageAccessor
         Task<IEnumerable<AvailableCoffeeCupModel>> GetCoffeeCupsAsync();
         Task<IEnumerable<BeverageSizeCostModel>> AddBeverageCostAsync(IEnumerable<BeverageSizeCostModel> prices);
         Task<IEnumerable<BeverageSizeCostModel>> GetBeverageCostAsync();
-        Task<IEnumerable<LineItemModel>> PurchaseAsync(IEnumerable<LineItemModel> lineItems);
+        Task<IEnumerable<InvoiceModel>> PurchaseAsync(IEnumerable<LineItemModel> lineItems);
```

Comment:
Remember, you will only ever need to return one invoive, not an IEnumerable of invoices